src/shared/ui/date-simple/date-simple.test.tsx

```
@@ -0,0 +1,108 @@
+    expect(labelSeparatorElement).toBeNull();
+  });
+
+  it('displays start date and additional info passed as children', () => {
+    const startDate = '2088-01-01';
+    const additionalInfoText = 'Test additional info element';
+
+    render(
+      <DateSimple startDate={startDate}>
+        <>
+          <span>{additionalInfoText}</span>
+        </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)